bridge/client/assets/default-branding/app-config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "appTitle": "keptn",
+  "logoUrl": "assets/branding/logo.png",
+  "logoInvertedUrl": "assets/branding/logo_inverted.png"
```

Comment:
Is the property: `"stylesheetUrl": "assets/branding/style.css"` missing here?

Reply:
The `stylesheetUrl` is optional and for the default theme not necessary.